### DISCUSSION TITLE
.ai domains

### DISCUSSION THREAD

**Discussion by mabry1985** [vercel/community#1323] [Help] [answered]
### Summary

I purchased a couple .ai domains and would love to transfer these in. Is this possibility on the horizon?

**Comment by swarnava** [comment, marked as answer]
Hi,

We don't support purchasing or transferring `.ai` domain on Vercel yet.  You can still use Vercel without any compromises by using our Nameservers. That way, you should have the same experience as if the domain was bought with us in the day-to-day activity.

**Comment by IlyaShkurenko** [comment]
@swarnava do you know when this option will be added? We can't use vercel just because of this

**Reply by swarnava** [reply]
Hi. There is no plan on this yet. You should still be able to buy from thirf party registrar and point to Vercel.

**Reply by IlyaShkurenko** [reply]
> Hi. There is no plan on this yet. You should still be able to buy from thirf party registrar and point to Vercel.

So I can buy ai domain and then click button Add instead of transfer?

**Comment by AlbertMarashi** [comment]
Come on, vercel should really support .ai domain names. You have 2 entire product stacks dedicated to AI services

**PS**  @Vercel Team - many of us don't care about privacy protection on domains, we understand if anonymizing domains is not possible for certain domains (because they are ccTLDs) - a simple notice stating privacy protection is not available for ccTLDs is sufficient, we just want to be able to buy and manage our domains via vercel

**Reply by AlbertMarashi** [reply]
Fun Fact: `.ai` is a ccTLD for the Government of Anguilla and the revenues from `.ai` domains corresponds to **_10% of their national GDP_**

**Comment by MartinDawson** [comment]
massive AI boom. Tons of devs using .ai domains for their development.

2 years later = Still no .ai domains from vercel. Bizarre that this isn't priority.

**Reply by PyPilot-ai** [reply]
^^What he said

[a comment by an automated account is not shown]
